fast_plotter/postproc/__main__.py

```diff
@@ -86,9 +86,18 @@ def main(args=None):
 
     sequence = read_processing_cfg(args.post_process, args.outdir)
 
+    def eval_apply_if(df, stage):
```

Comment:
I added a new function here because `dfs` is a list of tuples, so the `apply_to` list on l95 needs to take in `df[0]` as input rather than `df`. The `df` isn't used explicitly in the function but is part of the string evaluated by `eval` in the config. By having it as a separate function, `df` can be called in the config whilst referring to what's actually `df[0]` (for df in dfs) in the script.

Reply:
It's nice to extract this bit into a separate function but in general you should avoid the pure python`eval` since it's typically a security risk (e.g. https://stackoverflow.com/questions/661084/security-of-pythons-eval-on-untrusted-strings). When you've encounter `eval` and `query in this package so far it's actually a function that pandas provides rather than pure python's `eval` method.  To know what we could do instead i need to see how you intend this function to be used so it would help to have an example config.

Reply:
Given that you include `df` as an argument to the function and from what you say in the comment above, it might be enough to swap to `df.eval(...)` but again that sort of depends on what sorts of operations you want to allow.